Makefile

```diff
@@ -1,24 +1,31 @@
 BUILD_VERSION := $(shell cat VERSION)
+BUILD_DATE := $(shell date '+%Y%m%d')
```

Comment:
Could it be worth adding a timestamp to this? It's not unfeasible that we'd have two builds in one day?

Reply:
So the date format is based on what other images on Docker hub use.

I think that the idea is that we shouldn't regularly be building images more than once a day; I think a 3 day cadence would be more than enough, but that's a question for later.

If we need to build an image more than once in one day then I think that means we should also be updating the version number, because either we've had to rebuild it because something broke and it needed fixing, or we're making new releases because we've added new features.

*tl;dr* I don't think a timestamp adds much

Reply:
I think we might not *intend* to release more than once a day, but the problem is that screwups happen.

Reply:
➕for timestamp. Otherwise looks sensible to me 👍